ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/AgingImages.tsx

```diff
@@ -69,10 +72,17 @@ function getWidgetTitle(
         return 'Aging images';
     }
 
-    const totalImages =
-        Object.values(timeRangeCounts).find((range, index) => {
-            return selectedTimeRanges[index].enabled;
-        }) ?? 0;
+    // The total number of images is obtained by finding the first enabled bucket
+    // and getting the sum of that bucket and all following buckets.
+    let totalImages = 0;
+    const firstEnabledIndex = selectedTimeRanges.findIndex(({ enabled }) => enabled);
+    const rangesToCount = Object.values(timeRangeCounts).slice(
+        firstEnabledIndex,
+        selectedTimeRanges.length
```

Comment:
Not important to merge, but for my understanding: can omit length argument, or not?

> If end is omitted, slice extracts through the end of the sequence (arr.length).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

Reply:
Oh, good catch!